### ISSUE TITLE
Typo in Bitwise Manipulation Explore Card, subchapter "The basics of bit manipulation"

### ISSUE THREAD

**Issue by JonMrowczynski** [LeetCode-Feedback/LeetCode-Feedback#17066]
#### Your LeetCode username
jonmrowczynski

#### Category of the bug
- [ ] Question
- [ ] Solution
- [x] Language
- [ ] Missing Test Cases 

#### Description of the bug
In the Bitwise Manipulation Explore Card, chapter "Bit Manipulation: Concepts", subchapter "The basics of bit manipulation", there is a typo as indicated in the attached screenshot.

#### Code you used for Submit/Run operation
N/A

#### Language used for code
N/A

#### Expected behavior
Basically, the "is" should be moved to where the arrow is pointing in the screenshot.

#### Screenshots
![Bitwise Manipulation Typo](https://github.com/LeetCode-Feedback/LeetCode-Feedback/assets/39063205/c1787e7b-4135-4e5b-b63d-e1ad7de5a44c)

#### Additional context
N/A

**Comment by LC-Pam** [comment]
Hi there,

Thank you for reaching out to us.

 We've relayed this issue to our team to investigate. 

We will reach back to you after we have the results.

Thank you for your support.

**Comment by LC-Pam** [comment]
Thank you for your time. 

We've used your feedback to update the problem. 

Your LeetCode account has received 100 LeetCoins as a reward for this feedback. 

If you have any other questions or feedback, please don't hesitate to let us know! 

We appreciate your support!
